lookout/values.yaml

```diff
@@ -2,44 +2,33 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 #
-deployDummyAnalyzer: true
-
 replicaCount:
-  lookout: 1
-  dummyAnalyzer: 1
+  watch: 1
+  work: 3
 
 image:
   lookout:
     repository: srcd/lookout
     #tag: latest
     pullPolicy: IfNotPresent
-  dummyAnalyzer:
-    repository: srcd/lookout-dummy-analyzer
-    tag: latest
-    pullPolicy: Always
 
 service:
-  lookout:
+  work:
     type: ClusterIP
     port: 10301
-  dummyAnalyzer:
-    type: ClusterIP
-    port: 10302
 
 app:
-  lookout:
-    port: 10301
+  lookout: # Shared between watchers and workers
     logFormat: json
     logLevel: info
     probesPort: 8090
-  # bblfshdConnectionString: ipv4://bblfshd-bblfshd:9432
-
-  # Do not set this and providers.github.private_key
-  # secretName:
-  dummyAnalyzer:
-    port: 10302
-    probesPort: 8091
-
+    queue: lookout
+    broker: amqp://lookout-rabbitmq-ha:5672
```

Comment:
will use `{{ .Release.name }}` here

Reply:
Update: i forgot this is in values, i can not use templates here. The other option is to not use a value here and set `{{ .Release.name }}-rabbitmq-ha` (rabbitmq-ha is the subchart name) instead of the value

Reply:
I guess it's better since we want to use the rabbitmq deployed with helm